ghauth.js

```diff
@@ -36,7 +36,9 @@ function createAuth (options, callback) {
     data = JSON.parse(data.toString())
 
     if (data.message)
```

Comment:
Came through in https://github.com/carldanley/ghauth/commit/f1945a3293f2a247dd82b0f5d2e9a6a643ab50b9.